_docs/tasks/rate-limiting.md

```diff
@@ -23,39 +23,88 @@ This task shows you how to use Istio to dynamically limit the traffic to a servi
+ 
+Consider `ratings` as an external paid service like Rotten Tomatoes® with `5qps` free quota.
+Using Istio we can ensure that 5qps is not breached.  
 
 1. Configure mixer with the rate limit:
 
    ```bash
-   # (TODO) istioctl create -f mixer-rule-ratings-ratelimit.yaml
-   kubectl apply -f ../../mixer-config-quota-bookinfo.yaml
+   istioctl mixer rule create global ratings.default.svc.cluster.local -f ratelimit.yml
    ```
+   where ratelimit.yml is 
+	```yaml
+	rules:
+	- aspects:
+	  - kind: quotas
+		 params:
+			quotas:
+			- descriptorName: RequestCount
+			  maxAmount: 5
+			  expiration: 1s
+
+	```
+   `istioctl` sets configuration for `subject=ratings.default.svc.cluster.local`
 
 2. Generate load on the `productpage` with the following command:
 
    ```bash
    while true; do curl -s -o /dev/null http://$GATEWAY_URL/productpage; done
    ```
    
-   If you now refresh the `productpage` (http://$GATEWAY_URL/productpage)
-   you'll see that while the load generator is running 
-   (i.e., generating more than 5 req/s), we stop seeing stars.
+   If you refresh the `productpage` (http://$GATEWAY_URL/productpage) while the load 
+   generator is running (i.e., generating more than 5 req/s), the traffic generated by
+   your browser will be rate limited. When `reviews` service is unable to access `ratings`
+   service you stop seeing stars on the UI.
+
+### Conditional Rate Limiting
+
+In the previous example we applied rate limit to the `ratings` service without regard
+to any other attributes. It is possible to conditionally apply rate limit based on attributes like
+the source of the traffic.
+
+The following configuration applies `5qps` rate limit only to version `v3` of `reviews`.
+
+1. Configure mixer with the conditional rate limit:
+
+	```bash
+	istioctl mixer rule create global ratings.default.svc.cluster.local -f ratelimit-conditional.yml
+	```
+	where ratelimit-conditional.yml is 
+	```yaml
+	rules:
+	- selector: source.labels["app"]=="reviews" && source.labels["version"] == "v3"  
+	  aspects:
+	  - kind: quotas
+		 params:
+			quotas:
+			- descriptorName: RequestCount
+			  maxAmount: 5
+			  expiration: 1s
+
+	```
+2. Generate load on the `productpage` with the following command:
+
+   ```bash
+   while true; do curl -s -o /dev/null http://$GATEWAY_URL/productpage; done
+   ```
+   
+   If you refresh the `productpage` (http://$GATEWAY_URL/productpage) while the load 
+   generator is running (i.e., generating more than 5 req/s), the traffic generated by
+   your browser will be rate limited. 
 
-## Understanding ...
+	Since `reviews-v3` service is unable to access `ratings` we stop seeing `red` stars on the UI.
 
-Here's an interesting thing to know about the steps you just did.
 
-## What's next
-* Learn more about [this](...).
-* See this [related task](...).
+### Dimensioned Rate Limits [WIP]
 
+## What's next
+You can define conditional rate limits using any number of attributes.
```

Comment:
What's next is a section that should contain links to other documents. So this isn't the right place for additional information on the current doc.